### PR TITLE
register kagari.is-a.dev

### PR DESCRIPTION
**Pull request by kagarifuyukawa**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [x] The file is in the `domains` folder and is in the JSON format.
- [x] The file's name is all lowercased and alphanumeric. 
- [x] You have completed your website. 
- [x] The website is reachable.  
- [x] The CNAME record doesn't contain `https://` or `/`.  
- [x] There is sufficient information at the `owner` field.

### CODE REVIEW

File: domains/fuyukawa.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "subdomain": "fuyukawa",
```
Comment (cupglassDEV) [inline domains/fuyukawa.json:2]: We dont have that in `is-a.dev`, and this domain **isnt** managed by Open Domains
```suggestion
```

**Review by ghost — COMMENTED** [review]
only one issue, remaining..

**Comment by cupglassDEV** [comment]
I said we dont have that, remove the `subdomain` key

its not Open Domains

File: domains/fuyukawa.json
Comment (cupglassDEV) [inline domains/fuyukawa.json:1]: Change the file to `kagari.json` too

**Review by ghost — COMMENTED** [review]
Only one issue remaining, again...

**Pull request merged**